bindings/python/slimt.cpp

```diff
@@ -77,7 +79,11 @@ class PyService {
     for (auto &handle : handles) {
       auto &future = handle.future();
       future.wait();
-      responses.push_back(std::move(future.get()));
+      Response response = future.get();
+      if (encoding == Encoding::UTF8) {
+        change_ranges_to_utf8(response);
+      }
```

Comment:
This `if` can be changed to dispatch.

```cpp
transform_in_place(response, encoding)
```

If not change, the early return will take care of it.

Reply:
https://github.com/jerinphilip/slimt/pull/38/commits/a1064196d2bfae4eb709ce8aa7c53bef021fe618